### ISSUE TITLE
Associate Test Case/Steps with documents

### ISSUE THREAD

**Issue by robnist** [haffo/hit-iz-tool#196]
Not sure who this belongs to; reassign.

1. Provide test case/test step information in documents
2. Low priority for now; do once everything else is complete

![image](https://cloud.githubusercontent.com/assets/14874904/11486586/0cc92ff8-9788-11e5-9867-fe535a16836b.png)

**Comment by Jungyubw** [comment]
Sandra,

I and Anusha will fix it. Please pass this ticket for me.
